store/storetest/status_store.go

```diff
@@ -33,7 +33,7 @@ func testStatusStore(t *testing.T, ss store.Store) {
 		t.Fatal(err)
 	}
 
-	if err := (<-ss.Status().Get(status.UserId)).Err; err != nil {
+	if _, err := ss.Status().Get(status.UserId); err != nil { //Fix
```

Comment:
Why the `//Fix` comment?

Reply:
That was there for my own reference, it appears I forgot to take it out. My apologies